angular/package.json

```diff
@@ -20,9 +20,11 @@
     "@angular/platform-browser": "^6.0.2",
     "@angular/platform-browser-dynamic": "^6.0.2",
     "@angular/router": "^6.0.2",
+    "bootstrap": "^4.1.3",
     "chart.js": "^2.7.2",
     "core-js": "^2.5.4",
     "ng2-charts": "^1.6.0",
+    "ng4-charts": "^1.0.2",
```

Comment:
are there now two version of the same library in there? Or do we still need the ng2-charts?

Reply:
good catch! ng2-charts is not needed anymore